bots/notify/src/main/java/org/openjdk/skara/bots/notify/issue/JbsBackport.java

```diff
@@ -49,9 +49,13 @@ private static URI backportRequest(URI uri) {
         }
     }
 
-    private Issue createBackportIssue(Issue primary) {
+    private Issue createBackportIssue(Issue primary, String fixVersion, String defaultSecurity) {
         var finalProperties = new HashMap<>(primary.properties());
         finalProperties.put("issuetype", JSON.of("Backport"));
+        finalProperties.put("fixVersion", JSON.of(fixVersion));
```

Comment:
Not sure I fully understand what this does in the context of this PR.

Reply:
You are right, that's not related to the actual issue, just something that I found missing in this code path. In practice, this path is only used in tests. When interacting with a real JBS instance, we have a backportRequest.